Util/AdminObjectAclManipulator.php

```diff
@@ -271,8 +271,21 @@ protected function buildForm(AdminObjectAclData $data, FormBuilderInterface $for
                     'attr'     => $attr,
                 );
             }
-
-            $formBuilder->add($key, new AclMatrixType(), array('permissions' => $permissions, 'acl_value' => $aclValue));
+            //@todo Remove when dropping Symfony <2.8 support
+            $aclMatrixType = new AclMatrixType();
+            if (method_exists($aclMatrixType, 'getBlockPrefix')) {
+                $formBuilder->add(
+                    $key,
+                    'Sonata\AdminBundle\Form\Type\AclMatrixType',
```

Comment:
The BC trick should be only here, with a ternary condition for example.

Not all the FormBuilder stuff should be put here.